cloud/user-management.md

```diff
@@ -0,0 +1,53 @@
+# User management
+You can manage users for your Timescale Cloud account by navigating to the
+`Members` section in your Cloud console. From here, you can see your FIXME
+
+## About Timescale Cloud user management
+Timescale Cloud allows you to collaborate with other users on your projects.
+When you have created your project, you can add other users so that they can see
+your project in their Timescale Cloud console. You can add, manage, and delete
+users from the `Members` section of the Cloud console.
+
+If you are the creator of the Project, you are assigned an Administrator role,
+and have the ability to add, manage, and delete other users. If you have been
+added to a project, you are assigned a Member role.
```

Comment:
Currently, Member roles also have the ability to add and delete other members, and change the project name. 

There isn't any huge difference between the roles for now, so I wonder if it should even be worth mentioning/distinguishing between them currently?